src/main/java/it/pagopa/interop/probing/eservice/registry/updater/util/RestClient.java

```diff
@@ -26,18 +26,21 @@ public static RestClient getInstance() throws IOException {
   }
 
   private static final String ESERVICE_OPERATION_URL = "api.operations.baseUrl";
+  private static final String ESERVICE_BASE_PATH = "api.eservice.basePath";
 
   private String eserviceOperationUrl;
+  private String eserviceBasePath;
 
   private RestClient() throws IOException {
     this.eserviceOperationUrl = PropertiesLoader.getInstance().getKey(ESERVICE_OPERATION_URL);
+    this.eserviceBasePath = PropertiesLoader.getInstance().getKey(ESERVICE_BASE_PATH);
```

Comment:
PropertiesLoader has to be a real singleton without a public onstructor and properties need to be centralized in a Map.